ambry-api/src/test/java/com.github.ambry/account/AccountContainerTest.java

```diff
@@ -245,22 +253,22 @@ public void testAccountBuilder() throws JSONException {
     // build an account with arguments supplied
     AccountBuilder accountBuilder = new AccountBuilder(refAccountId, refAccountName, refAccountStatus, null);
     Account accountByBuilder = accountBuilder.build();
-    assertAccountAgainstReference(accountByBuilder, false, false);
+    assertAccountAgainstReference(accountByBuilder, false, false, LATEST_CONTAINER_JSON_VERSION);
```

Comment:
I don't see this being tested for container schema version V1

Reply:
The test is now parameterized so we can test both versions.